projects/editions/examples/dfm.yml

```diff
@@ -0,0 +1,101 @@
+kernel:
+  image: "linuxkit/kernel:4.9.x"
+  cmdline: "console=ttyS0 console=tty0 page_poison=1"
+init:
+  - linuxkit/init:062e57b1d1e017e44c6339fc2b4cd41f3f10b2a9 # with memlogd
+  - linuxkit/runc:2649198589ef0020d99f613adaeda45ce0093a38
+  - linuxkit/containerd:cf2614f5a96c569a0bd4bd54e054a65ba17d167f
+  - linuxkit/ca-certificates:3344cdca1bc59fdfa17bd7f0fcbf491b9dbaa288
+  - linuxkit/memlogd:9b5834189f598f43c507f6938077113906f51012
+  - edtestorg/install-dfm:256c1e9a344ca9f68b4131759a4c8f868a3e790b # adds /dfm
+onboot:
+  - name: sysctl
+    image: "linuxkit/sysctl:2cf2f9d5b4d314ba1bfc22b2fe931924af666d8c"
+    net: host
```

Comment:
none of these config options are needed any more they are baked into the image label.